pyproject.toml

```diff
@@ -5,30 +5,33 @@ build-backend = "flit_core.buildapi"
 [project]
 name = "aiida-lammps"
 dynamic = ["version", "description"]
-authors = [{name = "Chris Sewell", email = "chrisj_sewell@hotmail.com"}, {name = "Jonathan Chico", email = "jonathan.chico@sandvik.com"}]
+authors = [
+    { name = "Chris Sewell", email = "chrisj_sewell@hotmail.com" },
+    { name = "Jonathan Chico", email = "jonathan.chico@sandvik.com" },
+]
 readme = "README.md"
-license = {file = "LICENSE"}
+license = { file = "LICENSE" }
 classifiers = [
     "Development Status :: 4 - Beta",
     "License :: OSI Approved :: MIT License",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
If you add this, maybe also add 3.11 to the test matrix of the CI? Also, please update `requires-python` to be `>=3.9`. Finally, do you also want to add 3.12 perhaps? aiida-core supports it

Reply:
Hi @sphuber !
Thanks for the comments, I wonder if adding the 3.12 is okay since well the plugin supports it but I have not found a lammps release candidate that can be installed with conda for python 3.12 (in general it is a bit of a pain to see which version of lammps is compatible with which python version since the only way seems to look into the conda wheels)

Reply:
Does lammps depend on Python?

Reply:
Only on the sense on how we install it, we install lammps using conda in the action, and then we use the same python version for aiida. So if we install install python 3.12 there is no lammps conda release for it.
We could install it in some other way, it is possible to do it via apt, but the versioning is a bit unclear there